### PR TITLE
Added `hint -t`, `hint -tc` and `hint -Jtc`, to open links in a new tab.

### PR DESCRIPTION
**Pull request by nfvs**
Using `-w newtab` wouldn't allow us to use selectors with `-c`, and using composite would always open a new empty tab on cancel (esc).

This simplifies the google/HN hinting results example:
`bindurl www.google.com F hint -Jtc .rc>.r>a`

### CODE REVIEW

**Comment by nfvs** [comment]
Also has the advantage of observing `tabopenpos`/`relatedopenpos` correctly, since the expected behavior is for new tabs from hinting to observe `relatedopenpos` (which `-b` already does), but without the `-t` flag we need to use `-W tabopen` which uses `tabopenpos`.

File: src/excmds.ts [hunk 1]
```diff
@@ -3487,7 +3488,7 @@ import * as hinting from "@src/content/hinting"
         - -A save-as the linked image
         - -; focus an element
         - -# yank an element's anchor URL to clipboard
-        - -c [selector] hint links that match the css selector
+        - -c* [selector] hint links that match the css selector
```
Comment (bovine3dom) [inline src/excmds.ts]: What does the star mean?

File: src/excmds.ts [hunk 2]
```diff
@@ -3473,6 +3473,7 @@ import * as hinting from "@src/content/hinting"
 /** Hint a page.
 
     @param option
+        - -t open in a new tab
```
Comment (bovine3dom) [inline src/excmds.ts]: We should probably say `open in a new foreground tab`

**Comment by bovine3dom** [comment]
LGTM. Will merge once we fix doc issues.

**Comment by nfvs** [comment]
Can't comment on the review apparently.

> What does the star mean?
I thought it meant it could be combined with other arguments, like -J and -c

File: src/excmds.ts (hunk 1 above)
Comment (bovine3dom) [reply src/excmds.ts]: I saw your reply by email but apparently can't see it here. I think it's obvious that the flag can be combined. I would interpret the * as being a literal part of the flag.

**Comment by nfvs** [comment]
Done and squashed into a single commit.

**Comment by bovine3dom** [comment]
Thanks!

**Pull request merged**